token-client/src/main/java/com/sap/cloud/security/xsuaa/client/XsuaaDefaultEndpoints.java

```diff
@@ -32,16 +33,32 @@
 	 *            - the base URI of XSUAA. Based on the base URI the tokenEndpoint,
 	 *            authorize and key set URI (JWKS) will be derived.
 	 * @deprecated gets removed with the major release 3.0.0 Use instead
-	 *             {@link #XsuaaDefaultEndpoints(OAuth2ServiceConfiguration)}
+	 *             {@link #XsuaaDefaultEndpoints(String, String)}
 	 */
 	@Deprecated
 	public XsuaaDefaultEndpoints(URI baseUri) {
 		assertNotNull(baseUri, "XSUAA base URI must not be null.");
-		LOGGER.debug("Xsuaa default service endpoint = {}", baseUri);
+		LOGGER.warn("Using deprecated constructor to initialize xsuaa default service endpoint = {}", baseUri);
 		this.baseUri = baseUri;
 		this.certUri = null;
 	}
 
+	/**
+	 * Creates a new XsuaaDefaultEndpoints.
+	 *
+	 * @param baseUri
+	 *            - the base URI of XSUAA. Based on the base URI the tokenEndpoint,
+	 *            authorize and key set URI (JWKS) will be derived. - the cert URI
```

Comment:
use @param certUri for the other argument

Reply:
good catch !